### PR TITLE
Move lookup element to the body element in height recalculation test

### PR DESCRIPTION
**Pull request by alexander-kotov-dx**
Fix tests after #10440

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 1]
```diff
@@ -2258,39 +2258,49 @@ QUnit.test("popover height should be recalculated after async datasource load(T6
         return;
     }
 
-    this.clock = sinon.useFakeTimers();
-    const items = ["item 1", "item 2", "item 3", "item 4"];
-    const instance = $("#lookup").dxLookup({
-        dataSource: new CustomStore({
-            load: function() {
-                var deferred = $.Deferred();
+    const $rootLookup = $("<div>").attr("id", "root-lookup").appendTo("body");
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]:
```suggestion
    const $rootLookup = $("<div>").appendTo("body");
```
?

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 2]
```diff
@@ -2258,39 +2258,49 @@ QUnit.test("popover height should be recalculated after async datasource load(T6
...
+    let instance;
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]:
```suggestion

```

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 3]
```diff
@@ -2258,39 +2258,49 @@ QUnit.test("popover height should be recalculated after async datasource load(T6
...
 
-                setTimeout(function() {
-                    deferred.resolve(items);
-                }, 500);
-
-                return deferred.promise();
+    try {
+        this.clock = sinon.useFakeTimers();
+        const items = ["item 1", "item 2", "item 3", "item 4"];
+        instance = $rootLookup.dxLookup({
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]:
```suggestion
        const instance = $rootLookup.dxLookup({
```

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 4]
```diff
@@ -2258,39 +2258,49 @@ QUnit.test("popover height should be recalculated after async datasource load(T6
...
+            usePopover: true,
+            opened: true
+        }).dxLookup("instance");
 
-    assert.ok($(instance.content()).height() >= $(instance.content()).find(".dx-scrollable-content").height());
-    this.clock.restore();
+        this.clock.tick(1000);
+        assert.ok($(instance.content()).height() >= $(instance.content()).find(".dx-scrollable-content").height(), $(instance.content()).height() + " >= " + $(instance.content()).find(".dx-scrollable-content").height());
+    } finally {
+        instance.dispose();
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]:
```suggestion

```
$rootLookup.remove(); - should lead to instance disposing.

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**